snap/hooks/configure

```diff
@@ -9,38 +9,15 @@ fi
 
 driver="$(snapctl get driver)"
 
-if [[ -n $driver && ! $driver =~ (LIBVIRT|QEMU) ]]; then
-    echo "\"$driver\" is not a supported virtualization driver"
-    exit 1
-fi
-
-driver_file="$SNAP_COMMON/driver"
-
-if [[ ! -v driver_saved ]]; then
-    driver_saved="$(cat $driver_file)" || true
-fi
-
-driver=${driver:-QEMU}
-driver_saved=${driver_saved:-QEMU}
-
 if [[ $driver != $driver_saved ]]; then
-    /snap/bin/multipass list &>/dev/null || /bin/true
-
-    if [ $? -ne 0 ]; then
-        echo "Cannot determine status of instances. Not switching driver."
-        exit 1
-    fi
-
-    if /snap/bin/multipass list --format=csv | grep -Ev '(\+--|Name)' | grep --quiet -iv 'Stopped'; then
-        echo "All instances need to be stopped."
-        echo "Please save any data and stop the instances before proceeding:"
-        echo
-        echo "multipass stop --all"
-        exit 1
-    fi
-
-    driver_saved=$driver
-    needs_restart=true
+    echo "`snap set` is no longer used for driver selection, please use:"
+    echo
+    echo "multipass set local.driver=$( echo $driver | awk '{print tolower($0)}' )"
+    exit 1
+elif [[ $driver_saved != "" ]]; then
+    multipass set local.driver=$( echo $driver_saved | awk '{print tolower($0)}')
```

Comment:
I get a failure installing the snap from this line. Error:

    error: cannot perform the following tasks:
    - Run configure hook of "multipass" snap if present (run hook "configure": /snap/multipass/x3/meta/hooks/configure: line 18: multipass: command not found)

I guess the full path is necessary